students/Ben_freeman/lesson02/series.py

```diff
@@ -0,0 +1,23 @@
+def sum_series(n,m=0,o=1):
+	"Starting with m and o, take the nth term of the sequence generated by recursively summing."
+	if n ==1:
+		return m
```

Comment:
make sure your editor is configured to only use 4 spaces (no tabs)

your current indentation is incorrect.

Reply:
Hello, sorry about the indentation errors! I checked on my end, and in my editor (Sublime Text) I was indenting with tabs instead of spaces, so hopefully that will not happen anymore. Thanks for the feedback.